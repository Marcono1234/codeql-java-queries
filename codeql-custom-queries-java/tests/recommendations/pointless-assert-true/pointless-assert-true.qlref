./recommendations/pointless-assert-true.ql
