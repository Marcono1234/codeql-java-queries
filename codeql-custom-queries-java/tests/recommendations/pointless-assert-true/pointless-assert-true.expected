| Test.java:5:9:5:20 | stmt | Pointless assert statement |
| Test.java:6:9:6:22 | stmt | Pointless assert statement |
| Test.java:7:9:7:37 | stmt | Pointless assert statement |
